commit 8509174fed0137e4ed1363601fe9401faee52ae7 (refs/changes/66/766/3)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-21T14:18:40-07:00 (3 years ago)
    
    DOCKER-954 tweak for better output on this test assert
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/test/integration/api-build.test.js b/test/integration/api-build.test.js
index d037c7c..59d7955 100644
--- a/test/integration/api-build.test.js
+++ b/test/integration/api-build.test.js
@@ -16,6 +16,7 @@
  *       which are run separately (e.g. in nightly).
  */
 
+var format = require('util').format;
 var path = require('path');
 
 var tar = require('tar-stream');
@@ -118,7 +119,9 @@ test('api: build', function (tt) {
 
                 var output = result.body;
                 var hasLabel = output.indexOf('LABEL sdcdockertest=true') >= 0;
-                t.ok(hasLabel, 'output contains LABEL sdcdockertest=true');
+                t.ok(hasLabel, format(
+                    'output contains LABEL sdcdockertest=true: output=%j',
+                    output));
 
                 var hasSuccess = output.indexOf('Successfully built') >= 0;
                 t.ok(hasSuccess, 'output contains Successfully built');
